Modules/Campaign/Policies/CampaignPolicy.php

```diff
@@ -21,33 +21,46 @@ public function __construct()
     }
 
     /**
-     * Determine whether the user can view the post.
+     * Determine whether the user can view any campaigns.
      *
-     * @param \Modules\User\Entities\User  $user
-     * @param \Modules\Campaign\Entities\Campaign  $campaign
+     * @param \Modules\User\Entities\User $user
+     * @return mixed
+     */
+    public function viewAny(User $user)
+    {
+        return $user->role === 'brand';
+    }
+
+    /**
+     * Determine whether the user can view the campaign.
+     *
+     * @param \Modules\User\Entities\User $user
+     * @param \Modules\Campaign\Entities\Campaign $campaign
      * @return mixed
      */
     public function view(User $user, Campaign $campaign)
     {
-        return TRUE;
+        return $user->id === $campaign->user_id;
     }
 
+
     /**
-     * Determine whether the user can create posts.
+     * Determine whether the user can create campaigns.
      *
-     * @param \Modules\User\Entities\User  $user
+     * @param \Modules\User\Entities\User $user
      * @return mixed
      */
     public function create(User $user)
     {
-        return $user->role == 'brand';
+
+        return $user->role === 'brand';
```

Comment:
Instead of using string "brand" define constant("ROLE_BRAND" ) in User Entity and use constant.